### ISSUE TITLE
xWindowsProcess: Integration tests fails randomly

### ISSUE THREAD

**Issue by johlju** [PowerShell/xPSDesiredStateConfiguration#420]
The integration tests for xWindowsProcess fail to create log file, often, but sometimes the tests passes without a problem. 

```
Context Should start a new testProcess instance as running
      [+] Should not have a logfile already present 83ms
The 'Microsoft.PowerShell.Management' module was not imported because the 'Microsoft.PowerShell.Management' snap-in was already imported.
      [+] Should compile without throwing 490ms
      [+] Should be able to call Get-DscConfiguration without throwing 132ms
      [+] Should return the correct configuration 145ms
      [-] Should create a logfile 76ms
        Expected $true, but got $false.
        105:                 $pathResult | Should Be $true
        at Invoke-LegacyAssertion, C:\Program Files\WindowsPowerShell\Modules\Pester\4.3.1\Functions\Assertions\Should.ps1: line 188
        at <ScriptBlock>, C:\projects\xpsdesiredstateconfiguration\Tests\Integration\MSFT_xWindowsProcess.Integration.Tests.ps1: line 105
```

**Comment by johlju** [comment]
The same problem exist in https://github.com/PowerShell/PSDscResources/issues/76.

**Comment by johlju** [comment]
A potential fix for this was merged in PR #421.
